core/ui/TestCases/DefaultTemplate.tid

```diff
@@ -0,0 +1,22 @@
+title: $:/core/ui/testcases/DefaultTemplate
+
+\whitespace trim
+<$let
+	state={{{ [<qualify "$:/state/testcase">] }}}
+>
+<div class="tc-testcase-wrapper">
+<div class="tc-testcase-header">
+<h2><$transclude tiddler="Description" mode="inline"/></h2>
+</div>
+<div class="tc-testcase-panes">
+<div class="tc-testcase-source">
+<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -Description -ExpectedResult -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/Source"/>
```

Comment:
Add `-[prefix<state>]` to this filter? That way a new tab like `$:/state/testcase--590634553--590634553` doesn't get created after switching tabs.

Or maybe even more should be excluded with: `-[prefix[$:/state/]] -[prefix[$:/temp/]]`

Reply:
I agree. That's why my first suggestion was to exclude just the tab state tiddlers with `-[prefix<state>]`. Thinking from the perspective of using this widget in the tiddlywiki.com docs, it seems like > 99.9% of the time the tab state tiddler will be clutter as you say.

If a more aggressive filter like my second suggestion is made the default, then maybe some or all of the filter can be made configurable via a config tiddler somehow.

I know a custom template can be provided to achieve full customization, but it seems like wanting to customize just the filter will be a common use case.

Reply:
Thanks @btheado, you're quite right, `-[prefix<state>]` works well, I'll push that momentarily.